source/hpack.c

```diff
@@ -611,6 +632,7 @@ int aws_hpack_insert_header(struct aws_hpack_context *context, const struct aws_
 
     /* If for whatever reason this new header is bigger than the total table size, burn everything to the ground. */
     if (AWS_UNLIKELY(header_size > context->dynamic_table.max_size)) {
+        /* #TODO handle this. It's not an error. It should simply result in an empty table RFC-7541 4.4 */
```

Comment:
Is it hard to clear the table? What makes this a TODO?

Reply:
I wasn't touching the table code in this PR
I just traced my way in here and got spooked, so leaving a todo

Reply:
:ghost: